content/php/concepts/math-functions /terms/pow/pow.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'pow()'
+Description: 'Returns the number raised to the power of the exponent.'
+Subjects: 
+  - 'Web Development
+  - 'Code Foundations'
+Tags: 
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  -'learn-PHP'
+  -' paths/web-development
+--- 
+
+The **`pow()`** function returns the number raised to the power of the exponent.
+
+## Syntax 
+
+```pseudo 
+pow ( number $base, number $exp ) : number
+``` 
+
+The `pow()` function takes up to two parameters: 
+
+-`base` is the number to be raised.
+-`exp` is the power for a base to be raised. 
+
```

Comment:
In the syntax section, show the syntax and describe the use of the `**` operator since you mention it in the example.